mapeditor/main.cpp

```diff
@@ -13,6 +13,10 @@
 
 int main(int argc, char * argv[])
 {
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+	QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+#endif
+
```

Comment:
Not against it, but why we only do this for Qt 5?

Reply:
in Qt 6 it's a warning and CI would fail due to -Werror

Reply:
Deprecation warning?

Reply:
This option is default in QT6 and cannot disabled.